### PR TITLE
fix: cli login auto close timeout

### PR DESCRIPTION
**Pull request by xiaolang124**
1. Fix CLI login problem when successfully login but command hangs for a while.

### CODE REVIEW

File: packages/cli/src/constants.ts [hunk 1]
```diff
@@ -76,5 +76,7 @@ export enum CLILogLevel {
 export const sqlPasswordQustionName = "sql-password";
 
 export const sqlPasswordConfirmQuestionName = "sql-confirm-password";
-
 export const deployPluginNodeName = "deploy-plugin";
+
+export const connection = "connection";
```
Comment (swatDong) [inline packages/cli/src/constants.ts:81]: "connection" and "close" are built-in event type and is defined by node-http APIs, ok to directly use without defining constants... just for code simplify.

``` node
// net.d.ts
on(event: "close", listener: () => void): this;
on(event: "connection", listener: (socket: Socket) => void): this;
```

File: packages/cli/src/commonlib/azureLogin.ts [hunk 2]
```diff
@@ -256,6 +256,7 @@ export class AzureAccountManager extends login implements AzureAccountProvider {
   }
 
   private async doGetAccountCredentialAsync(): Promise<TokenCredentialsBase | undefined> {
+    AzureAccountManager.codeFlowInstance.destroySockets();
```
Comment (swatDong) [inline packages/cli/src/commonlib/azureLogin.ts:259]: call `destroySockets` when `server.on("close")`

File: packages/cli/src/constants.ts (hunk 1 above)
Comment (xiaolang124) [reply packages/cli/src/constants.ts]: Remove inappropriate constants.

File: packages/cli/src/commonlib/azureLogin.ts (hunk 2 above)
Comment (xiaolang124) [reply packages/cli/src/commonlib/azureLogin.ts]: Use ```server.on("close")``` listener.

**Review by swatDong — APPROVED** [review]

**Pull request merged**